docs/design-proposals/apiserver/Data-Partition-Design.md

```diff
@@ -0,0 +1,198 @@
+---
+title: API Server Data Partition Design for CloudFabric
+authors:
+  - "@sindica"
+---
+
+# API Server Data Partition Design for CloudFabric
+
+## Table of Contents
+
+## Motivation
+
+Community version of Kubernetes can take up 5K nodes per cluster, each 
```

Comment:
Some small suggested changes:
- "can take up 5K nodes" --> "can take up to 5K nodes"
- "each nodes has" --> "each node has"
- I remember in Kubernetes each node can support 100 pods, instead of 30 pods?
- "CloudFabric" --> "Arktos"
- "With current community design **of** each api server **get**" --> "With current community design, each api server gets"
- "bottle necks" --> "bottlenecks"

Reply:
100 pods is individual limit. When there are 5K nodes, average 30 pods per node when the cluster can operate normally.
Fixed all other comments.